.github/workflows/test-connect.yml

```diff
@@ -69,7 +69,7 @@ jobs:
     outputs:
       dailyMatrix: ${{ steps.set-matrix-daily.outputs.dailyMatrix }}
       otherDevicesMatrix: ${{ steps.set-matrix-other-devices.outputs.otherDevicesMatrix }}
-      legacyCanaryFirmwareMatrix: ${{ steps.set-matrix-legacy-canary-firmware.outputs.legacyCanaryFirmwareMatrix }}
+      canaryFirmwareMatrix: ${{ steps.set-matrix-legacy-canary-firmware.outputs.canaryFirmwareMatrix }}
```

Comment:
we want to keep legacy tests. also please make a temporary commit that makes them run even I branch so that we are sure we did not break anything